src/Components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './TodoInfo.scss';
+import { Todo } from '../../Types/todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <div className="todo">
```

Comment:
```suggestion
export const TodoInfo: React.FC<Props> = ({ todo }) => (
    <div className="todo">
```
you can write without return